packages/n/nvidia-470-glx-driver/package.yml

```diff
@@ -162,6 +171,15 @@ install    : |
     install -D -m 00644 nvidia-settings.png $installdir/usr/share/pixmaps/nvidia-settings.png
     install -D -m 00644 nvidia.icd $installdir/usr/share/OpenCL/vendors/nvidia.icd
 
+    # Appdata
+    install -Dm00644 $pkgfiles/nvidia-470-current.metainfo.xml $installdir/usr/share/metainfo/nvidia-470-current.metainfo.xml
+    install -Dm00644 $pkgfiles/nvidia-470-lts.metainfo.xml $installdir/usr/share/metainfo/nvidia-470-lts.metainfo.xml
+    install -Dm00644 $pkgfiles/nvidia-470-32bit.metainfo.xml $installdir/usr/share/metainfo/nvidia-470-32bit.metainfo.xml
+    install -Dm00644 nvidia-settings.png $installdir/usr/share/pixmaps/nvidia-current.png
```

Comment:
My understanding is that `/usr/share/pixmaps` is deprecated and is only supported for backwards compatibility. Icons should generally be installed to `/usr/share/icons/hicolor/`, to the appropriate folder depending on png size. Preferably you should use a `.svg` icon, even if you have to ship it in the $pkgdir.